CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 - Update wet day frequency correction to incorporate method additions from Hempel et al 2013. (PRs #162 and #159, @dgergel)
+- floor and ceiling for DTR (PR #163 @emileten)
```

Comment:
Would you mind putting these two items under a `## Changed` heading here?

Reply:
Nevermind! I just added this heading to `main`. You will likely need to resolve the merge conflict though (sorry).

Marking resolved.